### PR TITLE
Users can comment on article

### PR DESCRIPTION
**Pull request by jyuw**
#### Description
Changes proposed in this pull request:
* ability to comment on articles

#### What I have learned working on this feature:
* nested routes, form withs

#### Screenshots:
screen shot 2018-03-15 at 8 55 27 pm

### CODE REVIEW

**Review by tochman — APPROVED** [review]
Looks good to me. Nicely scoped and short. Just the way I like it. The only comment I have is that you could bundle the 2 migrations together. At this stage (before being merged into the code base, it's perfectly fine. It's not a blocker though, so you get a green light on this one from me.

**Comment by tochman** [comment]
I like the title of this PR. Descriptive. Stick to that standard and ask your team to do so as well.

**Comment by Fleurtam** [comment]
Looks good!

**Pull request merged**
